src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Configuration/ApplicationInsightsSinkCorrelationOptions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using Arcus.Observability.Telemetry.Core;
+using GuardNet;
+
+namespace Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights.Configuration
+{
+    /// <summary>
+    /// User-defined configuration options to influence the behavior of the Azure Application Insights Serilog sink while assigning telemetry correlation.
+    /// </summary>
+    public class ApplicationInsightsSinkCorrelationOptions
+    {
+        private string _operationIdPropertyName = ContextProperties.Correlation.OperationId,
+                       _transactionIdPropertyName = ContextProperties.Correlation.TransactionId,
+                       _operationParentIdPropertyName = ContextProperties.Correlation.OperationParentId;
+
+        /// <summary>
+        /// Gets or sets the Serilog application property name where the correlation operation ID is set.
```

Comment:
Should we specifiy 'correlation' here ?
Isn't it sufficient and more clear to do this:

```suggestion
        /// Gets or sets the Serilog application property name where the operation ID is set.
```

Reply:
Do we need to explain what the OperationID is ?

Reply:
We could, but we should do this for all mentions of the correlation ID's. Maybe we could provide a link to the documentation site that does the explanation better than trying to do the same in code docs that nobody (?) reads 😄

Reply:
Will be added here: https://github.com/arcus-azure/arcus.observability/issues/431